macosx/PrefsController.mm

```diff
@@ -836,14 +840,57 @@ - (void)resetWarnings:(id)sender
     //[fDefaults removeObjectForKey: @"WarningLegal"];
 }
 
-- (void)setDefaultForMagnets:(id)sender
+- (IBAction)setDefaultForMagnets:(id)sender
```

Comment:
/note to self/
This way you can make action to be discoverable by Interface Builder and jump directly from code to the view that calls this action.